osl/interface_linux.go

```diff
@@ -91,6 +92,19 @@ func (i *nwIface) Routes() []*net.IPNet {
 	return routes
 }
 
+func (i *nwIface) IPAliases() []*net.IPNet {
+	i.Lock()
+	defer i.Unlock()
+
+	aliases := make([]*net.IPNet, len(i.addressAliases))
+	for index, alias := range i.addressAliases {
+		ipa := types.GetIPNetCopy(alias)
+		aliases[index] = ipa
```

Comment:
Agreed - This form was used to be consistent with the code just above : Routes(). Let me know which way you prefer.